### DISCUSSION TITLE
Confirmed - SGI Indigo HW

### DISCUSSION THREAD

**Discussion by henris42** [erichelgeson/BlueSCSI-v1#250] [General]
As stated, with latest 512 byte patches both HD and CDRom are success for these old Unix systems.

However working with CD sets seems a bit pain, see separate post in Ideas.

**Comment by erichelgeson** [comment]
Hi, it seems you're talking about v2 (referencing recent 512 byte patches) - any details you can add to the wiki (under the v2 repo) would be appreciated!
